chrome_manifest_v3/views/popupView.js

```
@@ -1055,16 +1070,15 @@ Gmail2Trello.PopupView.prototype.bindData = function (data) {
 
     const lastError_k = (self.lastError || '') + (self.lastError ? '\n' : '');
 
-    const dl_k = self.parent.deep_link; // Pointer to function for expedience
-    const data_k = dl_k(self, ['data']);
-    const newCard_k = dl_k(data_k, ['newCard']);
+    const data_k = self?.data || {};
+    const newCard_k = data_k?.newCard || {};
     let newCard = Object.assign({}, newCard_k);
     //// delete newCard.title;
     delete newCard.description;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)